exasol_integration_test_docker_environment/test/test_populate_data.py

```diff
@@ -66,11 +66,12 @@ def setUp(self) -> None:
     def _execute_sql_on_db(self, sql: str) -> str:
         with ContextDockerClient() as docker_client:
             print(f"Executing sql on db: '{sql}'")
-            test_container = docker_client.containers.get(self.environment.environment_info.
+            environment = self.environment # type: ignore
```

Comment:
A class attribute, maybe add a comment to the ignore

Reply:
Added a comment